src/components/AppFooter/AppFooter.tsx

```diff
@@ -3,33 +3,39 @@ import { Run } from '../../lib/run';
 import { DesktopHelp, MobileControls } from '../Help/Help';
 
 export type AppFooterProps = {
-  run: Run,
+  run: Run;
```

Comment:
Why are these semicolons? Shouldn't this be commas separating key-value pairs in an object?

This also points out that we don't have any testing on the code in this file. We should add tests for both of the components here.

Reply:
> Why are these semicolons? Shouldn't this be commas separating key-value pairs in an object?

The formatter seems to insist on them. Maybe someone thought type declarations should look like interface declarations?